### PR TITLE
Test with upstream compiler

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
me_right_now

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
Crazy

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ workflows:
         - test-2.070-prod
         - test-2.071-devel
         - test-2.071-prod
+        - test-2.077-devel
```
Comment (mihails-strasuns-sociomantic) [inline .circleci/config.yml]: I'd name it `test-2.077-upstream-{prod|devel}`, just to make it more obvious in reported github status

**Review by mihails-strasuns-sociomantic — COMMENTED** [review]
Just one small suggestion. Pretty cool that it just worked :)

**Comment by mathias-lang-sociomantic** [comment]
Updated

**Review by mihails-strasuns-sociomantic — APPROVED** [review]

**Comment by mihails-strasuns-sociomantic** [comment]
Thanks!

**Pull request merged**